src/client/pages/Offer/index.tsx

```diff
@@ -237,7 +241,34 @@ export const OfferPage = ({
           />
         )}
         <Perils offerData={offerData} />
-        <AppPromotionSection />
+        <AppPromotionSection
+          promotions={
+            hasCarInBundles
+              ? [
+                  {
+                    headingText: textKeys.OFFER_SWITCH_PROMOTION_HEADING_CAR(),
+                    bodyText: textKeys.OFFER_SWITCH_PROMOTION_PARAGRAPH_CAR(),
+                    imageUrl: '/new-member-assets/offer/app-promotion.jpg',
+                    imageAltText: 'App promotion',
```

Comment:
Is this image decorational? In that case it should have an empty alt text - doesn't seem like this adds much value to screen, no?

https://www.w3.org/WAI/tutorials/images/decorative/

Reply:
Ah, nice!